strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/AuthorizationConfigController.java

```diff
@@ -192,8 +192,8 @@ public ResponseEntity deleteRole(@ApiParam(value = "The name of the role", requi
     // Assign privileges to the anonymous user
 
     @RequestMapping(value = "anonymous/privileges",
-                    method = RequestMethod.POST,
-                    consumes = MediaType.APPLICATION_JSON_VALUE)
+            method = RequestMethod.POST,
```

Comment:
**Trivial/Cosmetic remarks:**

- There seems to be no change here and below (unless I'm not seeing it)...?  Could you please restore the old formatting?
- Also, method parameters should always be aligned to the first parameter like this:

      @RequestMapping(value = "anonymous/roles",
                      method = RequestMethod.POST,
                      consumes = MediaType.APPLICATION_JSON_VALUE)

(Both in methods and in annotations)